### PR TITLE
Add Swedn and VG clustering datasets

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
This pull request adds two new clustering datasets: SwednClustering and VGSummarizationClustering. It also includes performance tests for different example sizes.

### CODE REVIEW

File: src/seb/registered_tasks/mteb_tasks_clustering.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            documents, labels = list(zip(*pairs))
+
+            documents_batched = list(batched(documents, 512))[:4]
+            labels_batched = list(batched(labels, 512))[:4]
+            # (512x4) resampling changes scores from 12.68, 11.30, 12.65,
+            # (512x5) resampling changes scores from 11.5, 11.8,
+            # (2048x5) resampling changes scores from 11.40, 10.94, 10.95
+            # (5000x5) resampling changes scores from 10.9, 10.48
+            # conclusion: It seems like the more data gives a slightly better estimate, but not by much.
+            # it also seems like the large batches lead to a slightly lower score (I could imagine it is correlated with number of clusters).
```
Comment (KennethEnevoldsen) [inline src/seb/registered_tasks/mteb_tasks_clustering.py:161]: Did some of experimentation on size of clustering targets. 

@Muennighoff I could imagine you also have done some experiments here. I would love to hear what you have to say.

Of course, it is always nice to keep the dataset "lean" to allow folks to quickly experiment with models.

All of these experiments were with the all-mini-lm.

Comment (KennethEnevoldsen) [reply src/seb/registered_tasks/mteb_tasks_clustering.py:161]: See also #94

File: src/seb/registered_tasks/mteb_tasks_clustering.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+        pairs = list(zip(documents, labels))
+
+        rng = random.Random(42)  # local only seed
+        rng.shuffle(pairs)
+        documents, labels = list(zip(*pairs))
+
+        n_pairs = len(documents)  # 114k examples
+        n_splits = 10  # chosen semi-arbitrarily based on existing clustering tasks in MTEB  -
+        # we could also make sure that the summary, article and headlines are in the same bins?
```
[the remark this answers is not in the document]
Comment (Muennighoff) [reply src/seb/registered_tasks/mteb_tasks_clustering.py:85]: IIUC separate bins would be something like S2S vs P2P clustering in MTEB & if so I think it makes sense to put it into separate ones.

File: src/seb/registered_tasks/mteb_tasks_clustering.py (hunk 1 above)
Comment (Muennighoff) [reply src/seb/registered_tasks/mteb_tasks_clustering.py:161]: Yeah I'd keep it lean such that it doesn't take >15min to eval with all-mini-lm

File: src/seb/registered_tasks/mteb_tasks_clustering.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (KennethEnevoldsen) [reply src/seb/registered_tasks/mteb_tasks_clustering.py]: Yep. Moving it into separate bins also improved the performance quite a bit as well.

File: src/seb/registered_tasks/mteb_tasks_clustering.py (hunk 1 above)
Comment (KennethEnevoldsen) [reply src/seb/registered_tasks/mteb_tasks_clustering.py]: Perfect that is done now. I have also reduced the size of SwednClustering as well.

**Pull request merged**